meos/include/meos.h

```diff
@@ -1226,6 +1246,15 @@ extern TBox *intersection_tbox_tbox(const TBox *box1, const TBox *box2);
 extern STBox *union_stbox_stbox(const STBox *box1, const STBox *box2, bool strict);
 extern STBox *intersection_stbox_stbox(const STBox *box1, const STBox *box2);
 
+/*****************************************************************************
+ * RTree functions
+ *****************************************************************************/
+
+extern RTree * rtree_create (int basetype);
```

Comment:
Here I don't know what to do with the `meosType`. I consider is important to have this definitions in meos.h and not in meos_internal.h, but meosType is in meos_catalog.h. 

I went with this solution but feel free to recommend something else.

Reply:
I think we can probably just include `meos_catalog.h` in `meos.h`, but let me think about it.

Reply:
My problem with importing `meos_catalog.h` in `meos.h`  is definitions like `tempSubtype` which are done in `meos.h` are used in `meos_catalog.h`.

As far as I understand `meos.h` is the base for the rest of the definitions. So importing `meos_catalog.h` sounds weird. I was thinking in moving the definition of  `meosType` to `meos.h` but that should probably be another PR. WDYT?

Reply:
@estebanzimanyi Any thoughts on this?
We're trying to see how to define a function in `meos.h` which takes a `meosType` as input.

Reply:
For the moment let's leave it like this and we will see how to improve it later